spec/order_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'order'
+
+describe Order do
+  let(:menu) { Menu.new({ pizza: 4, burger: 3, hot_dog: 2 }) }
```

Comment:
Can implement a double here to isolate tests?  `let(:menu) { double("menu double") }`
And then allow menu to receive x method and return y